gpflow/kernels/mo_kernels.py

```diff
@@ -75,22 +82,26 @@ class SharedIndependent(MultioutputKernel):
     Note: this class is created only for testing and comparison purposes.
     Use `gpflow.kernels` instead for more efficient code.
     """
-    def __init__(self, kernel: Kernel, output_dimensionality: int):
+    def __init__(self, kernel: Kernel, num_latents: int):
```

Comment:
in this class `num_latents` equals `output_dimensionality`, so why are we changing this? It deprecates the shape comments as well. So I'm not a big fan of this.

Reply:
I think the best things to do here is to to have both `num_latent` and `output_dim` and show that they are the same.